Projects/Team1/Prystopchuk/Converter.cs

```diff
@@ -0,0 +1,69 @@
+﻿/*Method that converts a given number 
+ * from decimal to other base-N systems (N<=36)
+ * Return type: string*/
+
+namespace TEMP
+{
+    class Converter
+    {
+        
+        static string convert(int num, int base_, string res = "")
+        {
+            try
+            {
+                if (base_ <= 1 || base_ > 36) throw new System.Exception("Invalid base provided");
+                string alphabeth = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+                if (num < base_)
+                    return alphabeth[num].ToString();
+                else return convert(num / base_, base_) + alphabeth[num % base_];
```

Comment:
try to use ternary operator c# https://metanit.com/sharp/tutorial/2.5.php

Reply:
I've made changes and omg, i like it, thank you